src/styles/style.css

```diff
@@ -237,37 +213,6 @@ footer {
   }
```

Comment:
Hmmm we seem to have ended up with two `footer` sections (the other has a header `/* ---------- FOOTER ---------- */`)
Let's aggregate them

Reply:
Longer term we might want to split this CSS file - do you have any plans/thoughts in that direction?

Reply:
Good catch. I removed one. As for the CSS file to avoid this kind of stuff, 100% agree! I actually made a ticket about it this weekend. :)

https://github.com/nditcommunity/nditcommunity.github.io/issues/85